### PR TITLE
Update regress output for dither and large dither checks

### PR DESCRIPTION
**Pull request by jeanconn**
Update regress output for dither and large dither checks

### CODE REVIEW

File: test_regress/test/2014/JAN2514/oflsa/starcheck.txt [hunk 1]
```diff
@@ -0,0 +1,1310 @@
...
+[ 9]  5  1140596208   ACQ  6x6   5.797   9.005  10.516   -296   1086  20   1  120          
+[10]  6  1140726984   ACQ  6x6   5.797   9.008  10.516    184   -128  20   1  120          
+[11]  7  1140723840   ACQ  6x6   5.797   8.533  10.031    756   -320  20   1  120          
+[12]  0  1140722888   ACQ  6x6   5.797   8.327   9.828  -1409   -578  20   1  120          
+[13]  1  1140725600   ACQ  6x6   5.797   9.133  10.641   1946   -226  20   1  120    a2    
+[14]  2  1140724800   ACQ  6x6   5.797   9.328  10.828    546   -883  20   1  120    a2    
+
+>> WARNING: [ 5] Search spoiler. 1140594048: Y,Z,Radial,Mag seps:  71  94 118 -0.6
+>> WARNING: [14] Search spoiler. 1140726104: Y,Z,Radial,Mag seps:  41 117 124 -0.7
+>> WARNING: Non-standard dither
```
Comment (jeanconn) [inline test_regress/test/2014/JAN2514/oflsa/starcheck.txt]: Expected warning on standard amplitude but non-standard period.

File: test_regress/test/2014/JAN2514/oflsa/starcheck.txt [hunk 2]
```diff
@@ -0,0 +1,1310 @@
...
+[ 4]  3   171585880   BOT  6x6   5.797   8.580  10.078   -492    243  20   1  120          
+[ 5]  4   171586032   BOT  6x6   5.797   9.003  10.500    -98   1132  20   1  120          
+[ 6]  5   171597832   BOT  6x6   5.797   9.329  10.828   2304  -1598  20   1  120  a2g2    
+[ 7]  6   171721904   BOT  6x6   5.797   9.234  10.734   -152   2214  20   1  120          
+[ 8]  7   243941560   BOT  6x6   5.797   8.333   9.844  -1889    525  20   1  120          
+[ 9]  0   171586976   ACQ  6x6   5.797   8.424   9.922  -1393    908  20   1  120    a2    
+[10]  1   171721440   ACQ  6x6   5.797   9.376  10.875    112   1295  20   1  120    a2    
+[11]  2   171584504   ACQ  6x6   5.797   9.494  11.000    819    222  20   1  120    a2    
+
+>> WARNING: Non-standard dither
```
Comment (jeanconn) [inline test_regress/test/2014/JAN2514/oflsa/starcheck.txt]: Expected warning on 1 arcsec dither.

**Comment by taldcroft** [comment]
Needs rebase.

**Comment by jeanconn** [comment]
Haven't finished actually rerunning the tests.

**Comment by jeanconn** [comment]
Reran regression tests on rebased handle_big_dither branch, and updated and rebased this branch accordingly.

**Pull request merged**